### ISSUE TITLE
ID 258557: [Supporting the platform -Simon – Add to Dashboard]: "Cancel and Open in new tab" buttons under 'Add to Dashboard' blade are truncated after setting the viewport to 320*256.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66371]
### **User Experience:**  
Low vision who depends on zooming the page for better readability will be impacted as they will be able to see the complete content present in the page which will result in lack of understandability and will fail to access the controls.
_Note: User credentials should NOT be included in the bug._

### **Pre-Requisite:**
Achieve viewport 320 X 256 by adjusting the browser. (Open developer's tool, resize and zoom the browser to achieve viewport 320 X 256).

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org/) Sign in with valid credentials. Home page appears. 
2. Navigate to the left navigation bar and to the "Explore" button, select it. 
3. New "Explore" page appears, Navigate it. 
4. Navigate to the "Add to Dashboard" button and select it. 
5. Navigate the "Add to Dashboard" pop-up and apply changes. 
6. Verify the issue.

**Actual Result:** 
"Cancel and Open in new tab" buttons under 'Add to Dashboard' blade are truncated after setting the viewport to 320*256.

**Expected Result:**
After setting the viewport to 320*256, "Cancel and Open in new tab" buttons under 'Add to Dashboard' blade should be properly visible. Every control should be present on page, and it should be properly visible so users will navigate easily on the page.

### **Severity:**
`SEV2`

### **Area:**
_The same issue exists with list item of "Search by data source" combo box and info icon of 'Search by label' under 'Alert rules' tab. The same issue exists with info icon of 'Search by label' under 'Alert rules' tab. The same issue exists with "Explore" screen. (Home page>Explore)_ 
_Same issue appears for Feature showcases -> Share->Close_
_Related to the above issue, The RadioButtonGroup component is not responsive and causes horizontal scroll._

- [ ] #grafana-dashboards: `Share Modal`
- [ ] #alerting-experience: `MultipleDataSourcePicker Component` & `Query Editor`
- [ ] #grafana-frontend-platform: `RadioButtonGroup Component` & `Modal.ButtonRow Component`  (Saga Design System)
- [ ] #grafana-explore: `AddToDashboardModal Component`

**Comment by joshhunt** [comment]
In Explore specifically, Modal buttons don't wrap and are clipped off screen

![image](https://user-images.githubusercontent.com/46142/232007236-c1863175-c95f-434d-a127-67a84cec36ea.png)
